stc_game/Assets/Scripts/HUD/PotionButton.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public enum PotionType { HP, AP };
+
+public class PotionButton : MonoBehaviour
+{
+    public Transform parent;
+    public PotionType type;
+    public float cooldown;
+    public PotionCooldown potionCooldown;
+
+    private void Start()
+    {
+        potionCooldown = parent.GetComponentInChildren<PotionCooldown>();
+    }
+
+    public void UsePotion()
+    {
+        var stats = FindObjectOfType<Hero>().characterStats.stats;

```

Comment:
We'll need to make this into a Hero Controller reference down the road

Reply:
ok